tests/e2e/matmul/BUILD.bazel

```diff
@@ -61,6 +61,8 @@ py_binary(
 
 [iree_generated_trace_runner_test(
     name = "e2e_matmul_mmt4d_%s_large" % lhs_rhs_type,
+    # "--shapes=large" can cause timeouts on riscv emulator.
+    size = "large",
```

Comment:
I don't think we need to qualify all the places that large tests are large with comments like these. As useful as the context can be, I think a "--shapes=large" test with "large" in it's target name being labeled as "large" in size is pretty self-explanatory.